extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -134,42 +143,40 @@ public function getPages() {
 	 * Called by a view script to generate the advanced tab link in search.
 	 */
 	public function advancedTabLink() {
-		$term = $this->getVal('term');
-		$namespaces = $this->getVal('namespaces');
-		$label = $this->getVal('label');
-		$tooltip = $this->getVal('tooltip');
-		$params = $this->getVal('params');
+		$term = $this->getVal( 'term' );
+		$namespaces = $this->getVal( 'namespaces' );
+		$label = $this->getVal( 'label' );
+		$tooltip = $this->getVal( 'tooltip' );
+		$params = $this->getVal( 'params' );
 
 		$opt = $params;
 		foreach ( $namespaces as $n ) {
 			$opt['ns' . $n] = 1;
 		}
 
-		$stParams = array_merge( array( 'search' => $term ), $opt );
+		$stParams = array_merge( [ 'search' => $term ], $opt );
 
 		$title = SpecialPage::getTitleFor( 'Search' );
 
-		$this->setVal( 'class',     str_replace( ' ', '-', strtolower( $label ) ) );
-		$this->setVal( 'href',      $title->getLocalURL( $stParams ) );
-		$this->setVal( 'title',     $tooltip );
-		$this->setVal( 'label',     $label );
-		$this->setVal( 'tooltip',   $tooltip );
+		$this->setVal( 'class', str_replace( ' ', '-', strtolower( $label ) ) );
+		$this->setVal( 'href', $title->getLocalURL( $stParams ) );
+		$this->setVal( 'title', $tooltip );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)